chitterApi.test.js

```diff
@@ -0,0 +1,15 @@
+const ChitterApi = require('./chitterApi');
+require('jest-fetch-mock').enableMocks()
+
+describe('Chitter class', () => {
+  it('calls fetch and loads peeps', async () => {
+    const api = new ChitterApi();
+    fetch.mockResponseOnce(JSON.stringify({
+    
+    }));
+
+    api.loadPeeps((peeps) => {
+      expect(loadPeeps.peeps.size).toEqual(50);
```

Comment:
Your mock fetch does not have 50 peeps to start with. Remember your test can not actually access the fetch that is why we are using a mock fetch to run the test. Have to do something to the fetch response and then check if it it actually working.